src/ZoneServer/Buffs/Handlers/OverReinforce_Buff.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Melia.Shared.Tos.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Scripting;
+using Melia.Zone.Skills;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.Characters.Components;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Over Reinforce Buff, which increases the target's physical and magic attack
+	/// </summary>
+	[BuffHandler(BuffId.OverReinforce_Buff)]
+	public class OverReinforce_Buff : BuffHandler
+	{
+		private const string VarName = "Melia.MagicAndPhysicalAttackBonus";
+
+		public override void OnStart(Buff buff)
+		{
+			var target = buff.Target as Character;
```

Comment:
Is there a reason for these `Character` checks? If you remove them we could use this for monsters as well if we wanted to.